honeybee_energy/simulationparameter.py

```diff
@@ -286,22 +298,42 @@ def to_idf(self):
         objects that make up the SimulationParameter (ie. RunPeriod, SimulationControl,
         etc.),
         """
-        header_str = '!-   ==============  SIMULATION PARAMETERS ==============\n'
+        sim_param_str = ['!-   ==========================================\n'
+                         '!-   =========  SIMULATION PARAMETERS =========\n'
+                         '!-   ==========================================\n']
+
+        # add the outputs requested
         table_style, output_vars, reports, sqlite, surfaces = self.output.to_idf()
-        output_vars_str = '/n/n'.join(output_vars) if output_vars is not None else ''
+        sim_param_str.append(table_style)
+        if output_vars is not None:
+            sim_param_str.append('\n\n'.join(output_vars))
+        if reports is not None:
+            sim_param_str.append(reports)
+        if sqlite is not None:
+            sim_param_str.append(sqlite)
+        sim_param_str.append(surfaces)
+
+        # add simulation settings
+        sim_param_str.append(self.simulation_control.to_idf())
+        sim_param_str.append(self.shadow_calculation.to_idf())
+        sim_param_str.append(generate_idf_string(
+            'Timestep', [self.timestep], ['timesteps per hour']))
+
+        # add the run period
         run_period_str, holidays, daylight_saving = self.run_period.to_idf()
-        holiday_str = '/n/n'.join(holidays) if holidays is not None else ''
-        daylight_saving_time_str = daylight_saving if daylight_saving is not None else ''
-        timestep_str = generate_idf_string(
-            'Timestep', [self.timestep], ['timesteps per hour'])
-        sim_control_str = self.simulation_control.to_idf()
-        shadow_calc_str = self.shadow_calculation.to_idf()
-        sizing_par_str = self.sizing_parameter.to_idf()
-
-        return '/n/n'.join([table_style, output_vars_str, reports, sqlite, surfaces,
-                            header_str, sim_control_str, shadow_calc_str, timestep_str,
-                            run_period_str, holiday_str, daylight_saving_time_str,
-                            sizing_par_str])
+        sim_param_str.append(run_period_str)
+        if holidays is not None:
+            sim_param_str.append('\n\n'.join(holidays))
```

Comment:
Wouldn't '\n' just do it in this case?

Reply:
It would work but it wouldn't look good in the resulting idf file. Essentially, there's be no blank line in between each object, making it look like one huge object.